addon-test-support/dom/find-tooltip-target.js

```diff
@@ -0,0 +1,18 @@
+export function findTooltipTarget(selector) {
+  if (!selector) { // In case of passing null, undefined, etc
+    selector = '.ember-tooltip-target, .ember-popover-target';
+  }
+
+  const { body } = document;
+  const tooltipTarget = body.querySelectorAll(selector);
+
+  if (tooltipTarget.length === 0) {
+    throw Error('ember-tooltips/test-support/dom/find-tooltip-target: No tooltip targets were found.');
```

Comment:
tiny style thing: I'd like to keep consistent with `throw new Error`, as we use elsewhere

Reply:
Ah, that was not on purpose, will fix this!

Reply:
BTW, I basically only copied this over from the existing test helpers, which use `throw Error`. But I do agree that `throw new Error` is nicer!